android/app/src/main/java/app/organicmaps/downloader/DownloaderAdapter.java

```diff
@@ -387,40 +387,24 @@ private void processClick(boolean clickOnStatus)
     {
       switch (mItem.status)
       {
-      case CountryItem.STATUS_DONE:
-      case CountryItem.STATUS_PROGRESS:
-      case CountryItem.STATUS_APPLYING:
-      case CountryItem.STATUS_ENQUEUED:
-        processLongClick();
-        break;
-
-      case CountryItem.STATUS_DOWNLOADABLE:
-      case CountryItem.STATUS_PARTLY:
-        if (clickOnStatus)
-          onDownloadActionSelected(mItem, DownloaderAdapter.this);
-        else
-          processLongClick();
-        break;
-
-      case CountryItem.STATUS_FAILED:
-        RetryFailedDownloadConfirmationListener listener =
-            new RetryFailedDownloadConfirmationListener(mActivity.getApplication());
-        MapManager.warn3gAndRetry(mActivity, mItem.id, listener);
-        break;
-
-      case CountryItem.STATUS_UPDATABLE:
-        MapManager.warnOn3gUpdate(mActivity, mItem.id, new Runnable()
+        case CountryItem.STATUS_DONE, CountryItem.STATUS_PROGRESS, CountryItem.STATUS_APPLYING, CountryItem.STATUS_ENQUEUED ->
```

Comment:
Is it possible to import CountryItem, and use `STATUS_DONE` instead of `CountryItem.STATUS_DONE`?

Reply:
Yes by using static import
`import static app.organicmaps.downloader.CountryItem.*;`